### ISSUE TITLE
Incorrect Vertex Input in void GraphicsAPI_D3D11::SetPipeline(void *pipeline)?

### ISSUE THREAD

**Issue by emaschino** [KhronosGroup/OpenXR-Tutorials#120]
Hi,

I think that `element.SemanticIndex` is incorrectly set to `attribute.bindingIndex`. Shouldn't it be rather set to `attribute.attribIndex`?

```
diff --git a/Common/GraphicsAPI_D3D11.cpp b/Common/GraphicsAPI_D3D11.cpp
index 5573e91..954848f 100755
--- a/Common/GraphicsAPI_D3D11.cpp
+++ b/Common/GraphicsAPI_D3D11.cpp
@@ -761,7 +761,7 @@ void GraphicsAPI_D3D11::SetPipeline(void *pipeline) {
         for (const VertexInputAttribute &attribute : pipelineCI.vertexInputState.attributes) {
             D3D11_INPUT_ELEMENT_DESC element{};
             element.SemanticName = attribute.semanticName;
-            element.SemanticIndex = attribute.bindingIndex;
+            element.SemanticIndex = attribute.attribIndex;
             element.Format = ToDXGI_FORMAT(attribute.vertexType);
             element.InputSlot = attribute.bindingIndex;
             element.AlignedByteOffset = (UINT)attribute.offset;
```

**Comment by rbessems** [comment]
Yeah that is right. https://github.com/KhronosGroup/OpenXR-Tutorials/pull/121

**Comment by rbessems** [comment]
Published v1.0.3 of the site fixes this issue.
